build/circle-test.sh

```diff
@@ -49,14 +51,14 @@ fi
 # better.
 if [ "${CIRCLE_TEST_REPORTS}" != "" ]; then
     if [ -f "${outdir}/test.log" ]; then
-	mkdir -p "${CIRCLE_TEST_REPORTS}/go"
-	${builder} go2xunit < "${outdir}/test.log" \
-	       > "${CIRCLE_TEST_REPORTS}/go/test.xml"
+    mkdir -p "${CIRCLE_TEST_REPORTS}/go"
```

Comment:
The original file was using 4-space indents and 8-space tab characters. It looks like your editor replaced tabs with 4 spaces instead of 8.